src/app/meetups/add-meetups/meetups-add.component.ts

```diff
@@ -95,7 +105,8 @@ export class MeetupsAddComponent implements OnInit {
   }
 
   updateMeetup(meetupeInfo) {
-    this.couchService.put(this.dbName + '/' + this.id, { ...meetupeInfo, '_rev': this.revision }).subscribe(() => {
+    this.couchService.put(this.dbName + '/' + this.id, { ...meetupeInfo, '_rev': this.revision,
+     'startDate': this.getTimestamp(meetupeInfo.startDate), 'endDate': this.getTimestamp(meetupeInfo.endDate) }).subscribe(() => {

```

Comment:
date is converted to timestamp on saving but it is not converted to datestring on data patch